src/test/java/lotto/domain/LottoResultTest.java

```diff
@@ -18,7 +18,8 @@ public class LottoResultTest {
             )
         );
 
-        LottoResult lottoResult = lottoGroup.match(winLotto);
+        LottoWinningNumbers lottoWinningNumbers = new LottoWinningNumbers(winLotto, LottoNumber.valueOf(10));
```

Comment:
```suggestion
        LottoWinningNumbers lottoWinningNumbers = new LottoWinningNumbers(winLotto, 10);
```
부 생성자를 추가해 위와 같이 구현 가능하지 않을까?